stdlib/LinearAlgebra/src/dense.jl

```diff
@@ -1421,12 +1421,19 @@ function cond(A::AbstractMatrix, p::Real=2)
         return maxv == 0.0 ? oftype(real(A[1,1]),Inf) : maxv / minimum(v)
```

Comment:
For `A = zeros(Int, 2, 2)`, `cond(A, 2)` fails with an `InexactError`. I have changed this line to
```julia
return iszero(maxv) ? oftype(real(maxv), Inf) : maxv / minimum(v)
```